cmd/run.go

```diff
@@ -41,6 +41,11 @@ func (h *Handler) Run(ctx context.Context, req *entity.CommandRequest) error {
 		return errors.CommandNotSpecified
 	}
 
+	if _, err := exec.LookPath(req.Args[0]); err != nil {
+		fmt.Printf("%s is not in $PATH\n", req.Args[0])
+		os.Exit(1)
```

Comment:
This could probably be exit code 127, but that would make programs calling into `railway run` think `railway` is not found AFAIK.

Reply:
I think exit(1) is good here. We can change it later if need be. Thanks!